### PR TITLE
Added feature: Current active sounds and stop all button

### LINKED ISSUE #19: Current active sounds and stop all button
Right above the tiles and below the page title, show a message when at least 1 song is active:
"Sound(s) active: X. **Stop all.**"

- The bold part "Stop all." is a clickable link, which will stop all the current playing songs.
- The message when 0 songs play should not be visible
- The X in the message is showing the current playing songs.
- The font and design should be consistent with the rest of the website.

image

### PR DESCRIPTION
**Pull request by spookirs**
closes #19 

Please review. If you know a better way to stop all the sounds please let me know. Currently I'm using useEffect(), but I feel like that may not be the best way.

Thanks.

### CODE REVIEW

**Review by riccardobertolini — APPROVED** [review]

File: components/tilePlayer.tsx [hunk 1]
```diff
@@ -1,81 +1,101 @@
...
+  SliderContainer,
+  StyledSlider,
+} from './tilePlayer.style'
 
 interface TilePlayerProps {
-  src: string;
-  imageSrc: string;
+  src: string
+  imageSrc: string
+  incrementActiveSounds: (boolean) => void
```
Comment (riccardobertolini) [inline components/tilePlayer.tsx]: > 14 |   incrementActiveSounds: (boolean) => void

TS complains that here we should have an arguments, like
> 14 |   incrementActiveSounds: (value: boolean) => void

can you correct it please? :smile:

**Review by riccardobertolini — CHANGES_REQUESTED** [review]
check the comment :+1:

**Comment by riccardobertolini** [comment]
Thanks for correcting the problem :) from the preview looks great! However in order to merge it, you need to solve the conflicts

**Comment by spookirs** [comment]
@riccardobertolini should be good to merge now. :)

**Comment by riccardobertolini** [comment]
Great job! Thanks so much for your contribution

**Review by riccardobertolini — APPROVED** [review]

**Pull request merged**